### ISSUE TITLE
On a very narrow screen with very big text, the top nav bar obscures some of the question content

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#1030]
The `#questionContainer` element has `margin-top: 51px`, which is supposed to account for the size of the floating navbar. When the navbar is bigger than that because the text has to break to several lines, it obscures some of the question content.

This screenshot shows a question in a window half the width of my 1920×1080 screen, zoomed in 200%.

![image](https://github.com/numbas/Numbas/assets/19513/4488b6f1-1b49-4252-9d39-bb5194d2b15e)

The layout should be achieved with `display:grid` instead of absolutely-positioned elements, so that they can't overlap.
